challenge_1.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+    <head></head>
+    <title></title>
```

Comment:
You can add title to your html page, which is usually displayed in the browser's title bar (at the top). It is also displayed in browser bookmarks and search results.